packages/flutter/lib/src/painting/shader_warm_up.dart

```diff
@@ -85,15 +87,19 @@ abstract class ShaderWarmUp {
     final ui.Canvas canvas = ui.Canvas(recorder);
     await warmUpOnCanvas(canvas);
     final ui.Picture picture = recorder.endRecording();
+    assert(debugCaptureShaderWarmUpPicture(picture));
     if (!kIsWeb) { // Picture.toImage is not yet implemented on the web.
```

Comment:
Is this still true?

Reply:
No, but let's make that change in a separate PR? I'm not sure if the HTML renderer covers this sufficiently.

Reply:
the other thing is whether the HTML renderer would even benefit from this. This is meant to force Skia to compile shaders. I'm not sure how/if that does anything on the CanvasKit backend, but it probably does nothing on the HTML...